core/test/utils/index.js

```diff
@@ -728,20 +728,22 @@ setup = function setup() {
     };
 };
 
-// ## Functions for Route Tests (!!)
+// ## Shared functions for Routing Tests
 
 /**
  * This function manages the work of ensuring we have an overridden owner user, and grabbing an access token
  * @returns {deferred.promise<AccessToken>}
  */
 // TODO make this do the DB init as well
 doAuth = function doAuth() {
-    var options = arguments,
-        request = arguments[0],
-        fixtureOps;
+    let API_URL = arguments[0];
+    let request = arguments[1];
+    let options = arguments;
+    let fixtureOps;
 
-    // Remove request from this list
+    // Remove API_URL & request from this list
     delete options[0];
+    delete options[1];
```

Comment:
Instead of doing deletes on options, maybe more readable way would be doing:
```
    const API_URL = arguments[0];
    const request = arguments[1];
    let options = arguments.slice(2)
    // no need for delete operations here
```

Reply:
arguments is an object.

Reply:
right! so could be `Array.prototype.slice.call(arguments, 2)`?

Reply:
Could do yeah, not rly important i think? But will update 👍

Reply:
Updated